pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/SegmentPreProcessor.java

```diff
@@ -180,40 +181,42 @@ public boolean needProcess()
         DefaultColumnHandler defaultColumnHandler = DefaultColumnHandlerFactory
             .getDefaultColumnHandler(null, _segmentMetadata, _indexLoadingConfig, _schema, null);
         if (defaultColumnHandler.needUpdateDefaultColumns()) {
-          LOGGER.info("Found default columns need updates");
+          LOGGER.info("Found default columns need updates in segment: {}", _segmentMetadata.getName());
```

Comment:
(nit) can we just not log the name once at the entry point of the function ? That way we don't have to insert this into every LOGGER stmt inside if-else etc.

Reply:
That might not be sufficient to identify the exact segment for a log line if lots of segments are being loaded in parallel at startup right? Alternatively, we could implement [Mapped Diagnostic Context](https://www.slf4j.org/api/org/slf4j/MDC.html) but maybe that's overkill for just this use case.